### DISCUSSION TITLE
How to run mirthconnect server in background

### DISCUSSION THREAD

**Discussion by sal4455** [nextgenhealthcare/connect#5195] [Q&A]
Hi,
As mirthconnect mcserver is a foreground  process , so is there any way to run it in background ?

**Comment by pacmano1** [comment]
The standard installer natively creates a service in windows or background process on linux that start mirth server on reboot.   Those use mcservice, not mcserver.   Perhaps you should describe how you installed mirth.  If I recall correctly you posted before about controlling mirth via python.

Since you are new to mirth, I'd suggest on a clean workstation or virtual machine installing Mirth using the normal installer for your operating system using the defaults.   That's what most people do when they get started with Mirth.
